services/elevate-service

```diff
@@ -4,7 +4,7 @@ set -e
 SERVICE_NAME="${1:-org.webosbrew.hbchannel.service}"
 
 echo "[ ] Elevating '$SERVICE_NAME'..."
-sed -i 's;/usr/bin;/media/developer/apps/usr/palm/services/org.webosbrew.hbchannel.service;' /var/luna-service2-dev/services.d/$SERVICE_NAME.service
+sed -i "s;/usr/bin;/media/developer/apps/usr/palm/services/${SERVICE_NAME};" "/var/luna-service2-dev/services.d/${SERVICE_NAME}.service"
```

Comment:
This was hardcoded somewhat intentionally to allow for non-hbchannel app to be elevated to root. This effectively replaces `/usr/bin/run-js-service` to `/media/developer/apps/usr/palm/services/org.webosbrew.hbchannel.service/run-js-service`, and the latter path is not really guaranteed to exist in some other hypothetical `$SERVICE_NAME`

Reply:
Thanks for the explanation. Reverted.